src/nupic/algorithms/SpatialPooler.cpp

```diff
@@ -574,7 +575,7 @@ void SpatialPooler::initialize(vector<UInt> inputDimensions,
 
   tieBreaker_.resize(numColumns_);
   for (UInt i = 0; i < numColumns_; i++) {
-    tieBreaker_[i] = 0.01 * rng_.getReal64();
+    tieBreaker_[i] = pow(10, -1*(getPrecision()+1)) * rng_.getReal64(); // (PRECISION+1)-th place
```

Comment:
I don't think we should make this change. The tie breaker just has to distinguish between whole integer values here: https://github.com/breznak/nupic.core/blob/sp_init_params/src/nupic/algorithms/SpatialPooler.cpp#L1118

I actually think it might be better to drop all of the tie breaker state and randomly choose winners in the case of ties each time they occur. That would prevent one column from always getting preference over another.